OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -82,8 +82,15 @@ public interface INotifyBuildComplete { void BuildingComplete(Actor self); }
 	[RequireExplicitImplementation]
 	public interface INotifyDamageStateChanged { void DamageStateChanged(Actor self, AttackInfo e); }
 
+	[RequireExplicitImplementation]
+	public interface INotifyRepair
+	{
+		void StartRepairing(Actor self, Actor target);
```

Comment:
To be consistent with our other notification interfaces i'd like to see these renamed to `BeforeRepair`, `RepairTick` and `AfterRepair`.